src/main/java/coupon/domain/IssuedCoupon.java

```diff
@@ -0,0 +1,40 @@
+package coupon.domain;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class IssuedCoupon {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne
+    private Coupon coupon;
```

Comment:
```suggestion
    @ManyToOne
    private Coupon coupon;
```

하나의 쿠폰은 여러명에게도 발급 가능하지 않나요?

Reply:
앗, 이 필드는 쿠폰 필드입니다...!
발행된 쿠폰이 여러개의 쿠폰일 가능성은 없을 것 같아요~

Reply:
앗 ㅋㅋㅋ 잘못 봤어요 😢